### PR TITLE
Detect deleted files as deleted, although dpkg reports md5sum changes

### PR DESCRIPTION
**Pull request by tgoettlicher**

### CODE REVIEW

File: lib/rpm_database.rb [hunk 1]
```diff
@@ -26,6 +26,10 @@ def package_for_file_path(file)
     [package_name, package_version]
   end
 
+  def determine_changes(line)
```
Comment (mauromorales) [inline lib/rpm_database.rb]: can this be done with an alias?

File: lib/dpkg_database.rb [hunk 2]
```diff
@@ -45,6 +45,21 @@ def handle_verify_fail(_path)
     # This method is a no-op for DpkgDatabase.
   end
 
+  def determine_changes(line)
```
Comment (mauromorales) [inline lib/dpkg_database.rb]: is it possible to just have a parse_changes_line method for dpkg that does this logic and then calls super?

File: spec/unit/dpkg_database_spec.rb [hunk 3]
```diff
@@ -68,6 +68,16 @@
     end
   end
 
+  describe "#determine_changes" do
```
Comment (mauromorales) [inline spec/unit/dpkg_database_spec.rb]: if the method can be renamed then this could just be a context scenario inside the #parse_changes_line

**Comment by tgoettlicher** [comment]
Superseded by https://github.com/SUSE/machinery/pull/1735

**Pull request closed without merging**